www/docs/providers/credentials.md

```diff
@@ -39,6 +39,8 @@ The Credentials provider is specified like other providers, except that you need
 
   If you throw an Error, the user will be sent to the error page with the error message as a query parameter. If throw a URL (a string), the user will be redirected to the URL.
 
+Credentials provider `authorize()` method also provides request object as second parameter `req` (see example below).
```

Comment:
```suggestion
The Credentials provider's `authorize()` method also provides the request object as the second parameter (see example below).
```

Reply:
Found some additional places where the docs should be updated:
- https://next-auth-git-fork-igordanchenko-feat-add-request-t-297b00.vercel.app/providers/credentials#example-code
- https://next-auth-git-fork-igordanchenko-feat-add-request-t-297b00.vercel.app/tutorials/ldap-auth-example